CODE_OF_CONDUCT.md

```diff
@@ -1,4 +1,44 @@
 # Code of Conduct
```

Comment:
@ConorDavenport this one should be left as is, with the link over to the nodejs/admin repo

Reply:
Oh right, thanks for pointing that out